### ISSUE TITLE
Is this normal?

### ISSUE THREAD

**Issue by JesusMaxim** [HuberHaYu/Mi-HyperOS-AudioIO#15]
Redmi Note 13 Pro (4G)
HyperOS - 1.0.7.0 (October Patch)

[]()https://github.com/user-attachments/assets/b76299d3-281d-4299-aa85-b4d032808809

**Comment by HuberHaYu** [comment]
Yep, it is shown in the installation log, but it has not been translated into English yet.
